pbft/support/scripts/build.sh

```diff
@@ -3,11 +3,18 @@ set -x
 
 export GOOS=$1
 export GOARCH=amd64
+export PROJECTPATH="$GOPATH/src/github.com/truechain/truechain-consensus-core/pbft"
+
+cd $PROJECTPATH
 
 git_commit_hash() {
     echo $(git rev-parse --short HEAD)
 }
 
+protoc -I src/pbft-core/fastchain/ \
+          src/pbft-core/fastchain/fastchain.proto \
+          --go_out=plugins=grpc:src/pbft-core/fastchain/
```

Comment:
need to add to README's precursor list about `'golang-googlecode-goprotobuf' to provide command 'protoc-gen-go'`. Wasn't installed by default..

Reply:
Yes. This needs work.